.github/workflows/main.yaml

```
@@ -26,6 +26,9 @@ jobs:
 
       - name : Npm Install
         run: npm install
+      
+      - name: Test and coverage
+        run: npm run jest --coverage
```

Why not
```suggestion
        run: npm run test
```
This command should rгn `vitest --coverage`